po/oc.po

```diff
@@ -0,0 +1,626 @@
+
+#: pdfarranger/metadata.py:42
+msgid "Title"
+msgstr "Títol"
+
+#: pdfarranger/metadata.py:43
+msgid "Subject"
+msgstr "Subjècte"
+
+#: pdfarranger/metadata.py:44
+msgid "Keywords"
+msgstr "Mots clau"
+
+#: pdfarranger/metadata.py:45
+msgid "Producer"
+msgstr "Productor"
+
+#: pdfarranger/metadata.py:46
+msgid "Creator"
+msgstr "Creator"
+
+#: pdfarranger/metadata.py:47
+msgid "Creator tool"
+msgstr "Logicial de creacion"
+
+#: pdfarranger/metadata.py:48
+msgid "Created"
+msgstr "Creacion"
+
+#: pdfarranger/metadata.py:49
+msgid "Modified"
+msgstr "Modificacion"
+
+#: pdfarranger/metadata.py:169
+msgid "Invalid date format. Input discarded."
+msgstr ""
+
+#: pdfarranger/metadata.py:198
+msgid "Edit properties"
+msgstr "Modificar las proprietats"
+
+#: pdfarranger/metadata.py:212
+msgid "Property"
+msgstr "Proprietat"
+
+#: pdfarranger/metadata.py:212
+msgid "Value"
+msgstr "Valor "
+
+#: pdfarranger/pdfarranger.py:261
+msgid ""
+"PDF Arranger is a small python-gtk application, which helps the user to "
+"merge or split pdf documents and rotate, crop and rearrange their pages "
+"using an interactive and intuitive graphical interface. It is a frontend for "
+"pikepdf."
+msgstr ""
+
+#: pdfarranger/pdfarranger.py:272
+msgid "Print the version of PDF Arranger and exit"
+msgstr "Afichar la version de PDF Arranger e quitar"
+
+#: pdfarranger/pdfarranger.py:280
+msgid "File(s) to open"
+msgstr "Fichièr(s) de dobrir"
+
+#: pdfarranger/pdfarranger.py:429
+msgid "All pages must have the same size."
+msgstr ""
+
+#: pdfarranger/pdfarranger.py:457
+msgid "All supported files"
+msgstr "Totes los fichièrs preses en carga"
+
+#: pdfarranger/pdfarranger.py:461
+msgid "PDF files"
+msgstr "Fichièrs PDF"
+
+#: pdfarranger/pdfarranger.py:469
+msgid "All files"
+msgstr "Totes los fichièrs"
+
+#: pdfarranger/pdfarranger.py:474
+msgid "Supported image files"
+msgstr "Fichièrs imatge preses en carga"
+
+#: pdfarranger/pdfarranger.py:694
+msgid "Rendering…"
+msgstr "Rendut en cors…"
+
+#: pdfarranger/pdfarranger.py:772
+msgid "untitled"
+msgstr "sens títol"
+
+#: pdfarranger/pdfarranger.py:919 pdfarranger/pdfarranger.py:928
+#: pdfarranger/pdfarranger.py:1029 pdfarranger/pdfarranger.py:1082
+msgid "_Cancel"
+msgstr "_Anullar"
+
+#: pdfarranger/pdfarranger.py:927
+msgid "Your changes will be lost if you don’t save them."
+msgstr "Vòstras modificacions seràn perdudas se las enregistratz pas."
+
+#: pdfarranger/pdfarranger.py:928
+msgid "Do_n’t Save"
+msgstr "E_nregistrar pas"
+
+#: pdfarranger/pdfarranger.py:928 pdfarranger/pdfarranger.py:1028
+#: data/menu.ui.h:4
+msgid "_Save"
+msgstr "_Enregistrar"
+
+#: pdfarranger/pdfarranger.py:937
+msgid "Discard changes and close?"
+msgstr ""
+
+#: pdfarranger/pdfarranger.py:938 data/menu.ui.h:45
+msgid "_Close"
+msgstr "_Tampar"
+
+#: pdfarranger/pdfarranger.py:943
+msgid "Save changes to “{}” before closing?"
+msgstr ""
+
+#: pdfarranger/pdfarranger.py:946
+msgid "Save changes before closing?"
+msgstr ""
```

Comment:
This and the following 5 strings are missing translations which might be especially important so people can make the right decisions and not lose work.

Reply:
Yes, will make a more complete PR later then!

Reply:
Ideally just add commits to this PR i.e. to your main branch. Thanks!